source/js/buyers-guide/components/creep-vote/creep-vote.jsx

```diff
@@ -66,12 +65,21 @@ export default class CreepVote extends React.Component {
   }
 
   showVoteResult() {
-    if (this.state.creepinessSubmitted && this.state.confidenceSubmitted) {
-      let view = "SignUp";
-      if (this.state.subscribed) {
-        view = "DidVote";
+    const {
+      creepinessSubmitted,
+      confidenceSubmitted,
+      voteCount,
+      subscribed
+    } = this.state;
+
+    if (creepinessSubmitted && confidenceSubmitted) {
+      if (voteCount == 2 || 3) {
```

Comment:
Not sure what this condition check is about. Are we checking if `voteCount` is equal to `2` or `3`? According to the originally ticket I thought we only needed to check it against `2`?

Anyways I think we can simply line 76 - 82 to

```jsx
this.setState({
  showNewsletter: voteCount === 2,
  didVote: true
});
```

And just a note to `if (voteCount == 2 || 3)` (for future usage)

1. always use `===` instead `==`
2. `voteCount == 2 || 3`: this actually gives you the number `3` back. If you wanna compare `voteCount` with the number 2 and 3. You should do 

```js
if (voteCount === 2 || voteCount === 3) { ... }
```

Reply:
Ah you're right, it is evaluating to 3. I'll fix that, but no, we want to make sure the newsletter [shows only after the 2nd and 3rd vote](https://github.com/mozilla/foundation.mozilla.org/issues/3943#issuecomment-560533493).

Reply:
Ah. I def missed that comment. Thanks for clarifying.